drivers/hmis/app/controllers/hmis_client/assessments_controller.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class HmisClient::AssessmentsController < ApplicationController
+  include AjaxModalRails::Controller
+  include ClientDependentControllers
+
+  before_action :require_can_view_enrollment_details_tab!
```

Comment:
This is the permission currently in use for being able to drill-down on other similar assessments.  Let me know if it should be something else.